doc/examples/jupyter_notebook/readme.md

```diff
@@ -0,0 +1,49 @@
+# Jupyter Notebook using versioned Pachyderm Data
+
+In the following example, we will explore attaching a Jupyter server to Pachyderm and manipulating data in Pachyderm's data versioning.  Specifically, we will attach our Jupyter server to three different repositories used in an example Pachyderm pipeline:
+
+- **trips** - This repo is populated with a daily file that records the number of bicycle trips recorded by NYC's citibike bike sharing company on that particular day (data from [here](https://www.citibikenyc.com/system-data)).
+- **sales** - This repo includes a single CSV file called `sales.csv`.  `sales.csv` is updated daily by a pipeline that processes each file in `trips` to calculate sales for the day.  Note, here we are using a fictional multiplier, $5/trip, to calculate daily "sales" (i.e., these are not actually the sales figures for citibike).
+- **weather** - This repo is populated daily with a JSON file representing the weather forecast for that day from [forecast.io](https://darksky.net/forecast/40.7127,-74.0059/us12/en).
+
+We attach to the trips and weather repos at commit number 30, which corresponds to the data versioned on 7/31/2016. We did this, because on 7/30/2016 and 7/31/2016 we saw a sharp drop in our sales, and we want to try and understand, interactively, why we might have seen this drop in sales.
```

Comment:
Maybe this blurs the line between example/blog post. But I'd really love a visual of the DAG ... then where the jupyter service 'attaches' to the DAG